### ISSUE TITLE
attoparsec upper bound

### ISSUE THREAD

**Issue by snoyberg** [nominolo/atto-lisp#20]
Just one more upper bound I've found:

```
attoparsec-0.13.0.1 depended on by:
- atto-lisp-0.2.2.1 (>=0.10 && < 0.13). No maintainer. @nominolo
```

**Comment by FranklinChen** [comment]
It would be nice to move up to attoparsec 0.13.

**Comment by nominolo** [comment]
Fixed in 0.2.2.2.
